dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -0,0 +1,772 @@
+
+ERROR_FAILED_TO_INSTALL_FIX=CWWKF1222E: The fix {0} could not be applied successfully.
+ERROR_FAILED_TO_INSTALL_FIX.explanation=An error occurred when applying the fix.
+ERROR_FAILED_TO_INSTALL_FIX.useraction=Find out the reason from the wlp/lib/fixes/fixinstall.log.
+
+ERROR_INVALID_PRODUCT_VERSION=CWWKF1224E: The feature {0} cannot be installed to {1} {2} because it only applies to version {3}.  Use the "find" action of the featureManager command to retrieve a list of features that are applicable to {1} {2}.
+ERROR_INVALID_PRODUCT_VERSION.explanation=The feature is not applicable to the current version of the installed product.
+ERROR_INVALID_PRODUCT_VERSION.useraction=Install features that are supported by the product version.
+
+ERROR_INVALID_PRODUCT_EDITION=CWWKF1225E: The feature {0} cannot be installed to {1} {2} edition because it only applies to the editions {3}. Use the "find" action of the featureManager command to retrieve a list of features that are applicable to {4} {5} edition. 
+ERROR_INVALID_PRODUCT_EDITION.explanation=The feature is not applicable to the edition of the installed product.
+ERROR_INVALID_PRODUCT_EDITION.useraction=Install features that are supported by the product edition.
+
+ERROR_INVALID_PRODUCT_INSTALLTYPE=CWWKF1226E: The {0} asset cannot be downloaded or installed with the featureManager command because the product was installed by {1}, and the asset applies only to {2} installations. Use the featureManager find action to retrieve a list of assets that you can download or install with the command.
+ERROR_INVALID_PRODUCT_INSTALLTYPE.explanation=The featureManager command cannot download or install the asset to your Liberty installation because the asset applies only to a different installation type.
+ERROR_INVALID_PRODUCT_INSTALLTYPE.useraction=Use the featureManager find action to retrieve a list of assets that you can download or install with the command. For more information, see the product documentation for the asset.
+
+ERROR_FAILED_TO_RESOLVE_FEATURE_FROM_DIR=CWWKF1227E: Unable to obtain the {0} feature from the {1} directory.
+ERROR_FAILED_TO_RESOLVE_FEATURE_FROM_DIR.explanation=The installer could not get the specified feature from the specified local directory.
+ERROR_FAILED_TO_RESOLVE_FEATURE_FROM_DIR.useraction=Ensure that the directory contains the feature and retry the operation.
+
+ERROR_UNABLE_TO_LOCATE_COMMAND_EXE=CWWKF1228E: Unable to locate the {0} executable after checking the following locations: {1}
+ERROR_UNABLE_TO_LOCATE_COMMAND_EXE.explanation=The installer could not locate the executable file
```

Comment:
Change "The installer could not locate the executable file" to "The installer did not locate the executable file." or to "The installer cannot locate the executable file." (replace "could not" and add a period)